scripts/machine-launch-script.sh

```diff
@@ -281,6 +281,8 @@ set -o pipefail
         verilator==4.034 \
         screen \
     )
+    # clang-format for driver coding style enforcement.
+    CONDA_PACKAGE_SPECS+=( clang-format )
```

Comment:
Did some more looking and found it.  They put it in the `clang-tools` package.  I'll ask about splitting it out into the clang-format package so that we don't have to install the rest of clang but in the meantime, we could:
```suggestion
    CONDA_PACKAGE_SPECS+=( clang-format clang-tools )
```